### ISSUE TITLE
Something wrong with Appveyor

### ISSUE THREAD

**Issue by rmcgibbo** [omnia-md/conda-dev-recipes#14]
It looks like there is something catastrophically wrong with Appveyor (git not working).

**Comment by jchodera** [comment]
GitHub has been having trouble the last few days.  Travis keeps failing for the same reason.

**Comment by jchodera** [comment]
Let me try retriggering the build again.

**Comment by rmcgibbo** [comment]
I think it's related to the `git` executable not working correctly on Appveyor. IIRC we saw that before, and switched most of the meta.yaml `url:` fields to pull over http instead of using git.

**Comment by jchodera** [comment]
Hm, that could be it.  You able to take a stab at that?

**Comment by rmcgibbo** [comment]
I'm not really sure what the cause is, or what would need to be done. Pulling from http worked when you were asking for the release tags, but idk about here since you want the git HEAD.

**Comment by jchodera** [comment]
We can `wget https://github.com/pandegroup/openmm/archive/master.zip` and then `unzip` it.

**Comment by rmcgibbo** [comment]
I guess. The one problem is that in that case, `simtk.openmm.version` will  be wrong and not contain the proper git revision, but I guess that's only a minor annoyance.

**Comment by jchodera** [comment]
Does [this thread](https://github.com/babun/babun/issues/84) have a workable solution?

**Comment by rmcgibbo** [comment]
The error messages don't look the same

```
subprocess.CalledProcessError: Command '['C:\\cygwin\\bin\\git.exe', 'clone', '--mirror', u'https://github.com/parmed/parmed.git', u'/cygdrive/c/Miniconda/conda-bld/git_cache/__github.com_parmed_parmed.git']' returned non-zero exit status 3328
```

**Comment by jchodera** [comment]
Look earlier [(line 98)](https://ci.appveyor.com/project/jchodera/conda-dev-recipes/build/1.0.97/job/yqsest7da0hvamcm#L98):

```
C:/cygwin/usr/libexec/git-core/git-remote-https.exe: error while loading shared libraries: ?: cannot open shared object file: No such file or directory
```

**Comment by jchodera** [comment]
That shared library error is presumably the reason the `subprocess` error occurs.

**Comment by rmcgibbo** [comment]
oh, maybe then.

**Comment by rmcgibbo** [comment]
Fixed by #16
